test/units/modules/network/dcnm/dcnm_module.py

```diff
@@ -0,0 +1,116 @@
+# (c) 2020 Red Hat Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+import json
+
+from units.modules.utils import AnsibleExitJson, AnsibleFailJson, ModuleTestCase
+from units.modules.utils import set_module_args as _set_module_args
+
+
+def set_module_args(args, ignore_provider=True):
+    if 'provider' not in args and not ignore_provider:
```

Comment:
*Should* be safe to remove lines 30 - 31.  They were only needed for nxos_* standalone modules.  If you remove it and see any errors let me know.

Reply:
Thought of that as I am not using it anyway, but, left it anyway for comments. Removing does not show any errors.